pkg/model/volumes.go

```diff
@@ -22,6 +22,11 @@ import (
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 )
 
+const (
+	cloudAndStagingDefaultVolumeSize = "2Gi"
```

Comment:
```suggestion
	cloudVolumeSize = "2Gi"
```

as staging is an internal env which is basically cloud, can we make shorter variable name which means the same?